### ISSUE TITLE
Cannot read properties of undefined, reading `type`

### ISSUE THREAD

**Issue by OlegGulevskyy** [Ephem/rollup-plugin-preserve-directives#5]
Cannot bundle using your plugin:
```
[preserve-directives] Cannot read properties of undefined (reading 'type')
file: vite/modulepreload-polyfill
error during build:
TypeError: Cannot read properties of undefined (reading 'type')
    at Object.transform (/Users/oleggulevskyy/Documents/dev/gpt-for-excel/node_modules/rollup-plugin-preserve-directives/dist/index.js:24:84)
    at file:///Users/oleggulevskyy/Documents/dev/gpt-for-excel/node_modules/rollup/dist/es/shared/node-entry.js:24395:40
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
Using Vite with React + React query + Rect Intl, where it seems like `react-intl` isn't bundling because of this issue you are fixing. However, your plugin fails with this error when I am trying to bundle. Any idea?

Thanks

**Comment by elliason** [comment]
I have the same error. I use this plugin in my Vite UI library.
```js
{
rollupOptions: {
            output: {
                exports: 'named',
                preserveModules: true,
                inlineDynamicImports: false,
            },
            plugins: [
                preserveDirectives.default(),
            ],
        },
}
```

**Comment by Ephem** [comment]
Thanks for reporting this! This should hopefully be fixed by #6 and is released in `0.1.1`. Would love if you could test it and report back if it fixed your problems or not. 😃

I'll close the issue for now.

**Comment by elliason** [comment]
Tested and now it works fine for me.
